README.md

```diff
@@ -14,6 +14,8 @@ Documentation
 
 [You can view a list of our capabilities and how to use the API provided here.](docs/README.md)
 
+[Here you can view a list of our limitations.](docs/limitations.md)
+
```

Comment:
Yeah I thought about that.  I kind of wonder if that is something a person showing up on the main page would want to look at right away.  I think the architectural decisions are mostly for developers right?

I don't mind adding it at all though.  What ya think?